event_sidecar/src/types/database.rs

```diff
@@ -203,6 +242,31 @@ pub struct UniqueConstraintError {
     pub error: sqlx::Error,
 }
 
+#[derive(Debug, Clone)]
+pub struct DatabaseInitializationError {
+    reason: String,
+}
+
+impl ToString for DatabaseInitializationError {
+    fn to_string(&self) -> String {
+        format!("DatabaseInitializationError: {:?}", self.reason)
+    }
+}
```

Comment:
I'd suggest implementing `std::fmt::Display` instead of `ToString`.

Reply:
👍